ojd_daps_skills/getters/download_public_data.py

```diff
@@ -1,17 +1,30 @@
 from ojd_daps_skills import PUBLIC_DATA_FOLDER_NAME, PROJECT_DIR
 
 import os
-
+import platform
+import zipfile
 
 def download():
 
     public_data_dir = os.path.join(PROJECT_DIR, PUBLIC_DATA_FOLDER_NAME)
 
+    if platform.system() == "Windows":
+      os.system(
```

Comment:
good to know!! I hope we've covered all operating systems now

Reply:
Linux, macos and windows... that about sums it up